docs/examples/eslint-plugin-test/README.md

```diff
@@ -25,7 +25,7 @@ This plugin is for x purpose.
 🎨 Set in the `stylistic` configuration.\
 ⌨️ Set in the `typescript` configuration.\
 🔧 Automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--fix).\
```

Comment:
Regarding:

> I was tempted to also update the link for fixable rules too, since there's an equivalent Fixable Rules heading in the same ESLint docs page. But I kind of like how the two links point to two places and encourage users to read both...!

I think we could fix this by having the two ESLint docs on fixability link to each other.

* https://eslint.org/docs/latest/use/command-line-interface#--fix
* https://eslint.org/docs/latest/use/core-concepts#rule-fixes

Ideally, one doc on a topic links to all the other docs on the same topic...

Reply:
Addressing:

* https://github.com/eslint/eslint/pull/18080